examples/WebView/ReactNativeWebView.php

```diff
@@ -14,11 +14,11 @@ class ReactNativeWebView extends \LivingMarkup\Module
 {
     public function onRender()
     {
-        if($this->args['initiate'] == "true" && !empty($this->args['object'])) {
+        if ($this->args['initiate'] == "true" && !empty($this->args['object'])) {
             $object = $this->args['object']; // escape JSON
```

Comment:
The $this->getArgByName('object') may be better to use here as it has a null check and will not bomb if not provided in the widget.